### ISSUE TITLE
Current version is outdated, recent builds fail

### ISSUE THREAD

**Issue by exploide** [flathub/us.zoom.Zoom#399]
The current version served on Flathub is 5.14.7.2928 which is outdated and Zoom refuses to log in on our instance for that reason. While there is already 5.14.10.3738 on master and open pull requests for newer versions, they fail building because of

```
ERROR   src.manifest: Failed to check archive krb5/krb5-1.20.1.tar.gz with AnityaChecker: Error downloading upstream source: 404, message='Not Found', url=URL('https://kerberos.org/dist/krb5/1.21/krb5-1.21.-.tar.gz')
```

EDIT: apparently the latter problem could be fixed in #395

**Comment by chaddy314** [comment]
The Test Build/Fix mentioned in #395 
```bash
$ flatpak install --user https://dl.flathub.org/build-repo/31262/us.zoom.Zoom.flatpakref
```
currently returns a 404 error code.

**Comment by exploide** [comment]
Since #395 is merged and a newer version is available, the issue is fixed for me. Hope the PR updating to the most recent Zoom version will be merged soon.

Closing...
